.github/workflows/rust.yml

```
@@ -35,19 +31,14 @@
           lfs: true
 
       - name: Install stable toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          override: true
-          components: clippy
+        uses: dtolnay/rust-toolchain@stable
+        components: clippy
```

actionlint (1.7.4)

unexpected key "components" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)